course/models.py

```diff
@@ -3,25 +3,26 @@
 from embed_video.fields import EmbedVideoField
 
 
-class Department(models.Model):
-    title = models.CharField(max_length=100)
-    department_code = models.IntegerField(
-        verbose_name='department name', default=0, unique=True)
-    blog = models.TextField(
-        max_length=500,
-        default="""Hundreds of teachers have gone
-        through our system and become qualified
-        teachers of information technology. Study with us today""")
+class Unit(models.Model):
+    unit_name = models.CharField(verbose_name='unit name', max_length=100)
+    unit_code = models.IntegerField(
+        verbose_name='unit code', default=0, unique=True)
+    student = models.CharField(max_length=100)
+    video_url = EmbedVideoField()
+    video_id = models.CharField(blank=False, max_length=32)
```

Comment:
I don't think the video fields, and departments are necessary for now. The relationships have been defined in the video and course models. You can make student and tutor fields a relationship.